### PR TITLE
Update for SChinese Translation

### PR DESCRIPTION
**Pull request by ZeMingoh233**

### CODE REVIEW

File: SuperNewRoles/Resources/Translate.csv [hunk 1]
```diff
@@ -308,8 +308,8 @@ FinalStatusOverKillerOverKill,Over Kill!,オーバーキル!,肢解
 FinalStatusSuicideWisherSelfDeath,Suicide,自殺,自杀
 FinalStatusMadmakerMisSet,Misnaming,指名誤爆,背刺
 FinalStatusRevenge,Revenge,復讐,仇杀
-FinalStatusHitmanKill,Mission accomplished.,任務遂行
-FinalStatusHitmanDead,Disposition.,処分
+FinalStatusHitmanKill,Mission accomplished.,任務遂行,遇刺
+FinalStatusHitmanDead,Disposition.,沉江
```
Comment (Kurato-Tsukishiro) [inline SuperNewRoles/Resources/Translate.csv:312]: Sorry.
The "処分" is stated as a translation of the Japanese.

Sorry for the confusing word!

**Review by Kurato-Tsukishiro — COMMENTED** [review]
Thank you for the translation.

I am sorry.
May I ask for one correction only?

File: SuperNewRoles/Resources/Translate.csv (hunk 1 above)
Comment (ZeMingoh233) [reply SuperNewRoles/Resources/Translate.csv:312]: Oh sorry. I made a mistake. My fault!
Should I make a new PR?

Comment (Kurato-Tsukishiro) [reply SuperNewRoles/Resources/Translate.csv]: We have confirmed!
Thank you!!!

**Review by Kurato-Tsukishiro — APPROVED** [review]
Thank you!!!
Approved!

**Pull request merged**